services/ymax-planner/src/resolver.ts

```diff
@@ -53,3 +55,24 @@ export const resolvePendingTx = async ({
   // An offer error will result in a throw
   await signingSmartWalletKit.executeOffer(action);
 };
+
+export const resolvePendingTxByInvocation = async ({
+  walletStore,
+  txId,
+  status,
+  proposal,
+}: ResolveTxParams) => {
+  !proposal || Fail`Unexpected proposal ${q(proposal)}`;
+
+  const offerArgs = {
```

Comment:
`offerArgs` seems like an odd name. There's no offer. might as well inline this record.